src/components/HomePage/homePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage = () => (
```

Comment:
It's a good practice to name the component files with PascalCase. Please, rename the file from 'homePage.tsx' to 'HomePage.tsx' 📁